ngraph/python/tests/test_onnx/test_backend.py

```diff
@@ -211,10 +210,8 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
     (xfail_issue_47323,
      "OnnxBackendPyTorchOperatorModelTest.test_operator_add_broadcast_cpu",
      "OnnxBackendPyTorchOperatorModelTest.test_operator_addconstant_cpu",
-     "OnnxBackendPyTorchOperatorModelTest.test_operator_add_size1_right_broadcast_cpu",),
-    (xfail_issue_47330,
-     "OnnxBackendNodeModelTest.test_mod_mixed_sign_float64_cpu",
-     "OnnxBackendNodeModelTest.test_min_float64_cpu"),
+     "OnnxBackendPyTorchOperatorModelTest.test_operator_add_size1_right_broadcast_cpu",
+     "OnnxBackendNodeModelTest.test_mod_mixed_sign_float64_cpu"),
```

Comment:
This one still fails even with the CPU plugin changes?

Reply:
That's right, with other error message, so I moved it to other xfail.

Reply:
Now both of them are passing.